test/e2e_main_test.ts

```diff
@@ -18,7 +18,7 @@ describe('toClosureJS', () => {
     const closure = toClosureJS(
         {sourceMap: true, experimentalDecorators: true} as ts.CompilerOptions,
         ['test_files/underscore/export_underscore.ts', 'test_files/underscore/underscore.ts'],
-        {isUntyped: false} as Settings, diagnostics);
```

Comment:
I think the unnecessary as was me - is that something we could get the linter to complain about?